lib/index.js

```diff
@@ -153,11 +153,11 @@ const PSEUDO_HEADERS = {
  * @returns {Object} Headers.
  */
 function extractPseudoHeaders(params, requestOptions) {
-  for(const key in Object.values(PSEUDO_HEADERS)) {
-    if(params[key]) {
+  Object.values(PSEUDO_HEADERS).forEach(key => {
```

Comment:
Why the switch to `forEach`? You can do this instead:

`for(const key of Object.values(PSEUDO_HEADERS))`

Reply:
I was using in before which was returning the index thanks for the note on using of.

Reply:
Corrected in https://github.com/digitalbazaar/http-signature-header/pull/13/commits/3fe21e3a4f9a37ea5ee8898ba1b5133a8b0ad576